infra/log/KalturaLog.php

```diff
@@ -189,7 +189,7 @@ class LogMethod
 	public function __toString()
 	{
 		$backtraceIndex = 3;
-		$backtrace = debug_backtrace();
+		$backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
```

Comment:
Since this is relevant for php 5.3.6 and up and on CentOs the version (unless compiled manually) is 5.3.3 I think you should change to 
$backtrace = @debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
This way the log file won't be full of NOTICES